boards/ESP8266_12.md

```diff
@@ -14,6 +14,15 @@ Features
 * Built-in Wifi
 * 1023 JS variables
 
+Not supported
+-------------
+The following features are not supported by Espruino on the ESP8266:
+- No hardware I2C, however, the software I2C works OK.
+- No hardware SPI (implementing that is on the to-do list), the software SPI works OK.
+- No DAC/PWM/digitalWrite: the esp8266 does not have a DAC and the one PWM it has is very
```

Comment:
Can you help me understand the no "digitalWrite" comment?  I thought we had the `digitalWrite` API as found here:

http://www.espruino.com/Reference#l__global_digitalWrite